src/client/components/Editors/Package/Sections/Package.tsx

```diff
@@ -118,14 +119,16 @@ function Version() {
   )
 }
 function Created() {
+  const created = useStore((state) => dayjs(state.descriptor.created))
```

Comment:
I think you need to keep it  as `const created = useStore((state) => state.descriptor.created)` -- useStore will re-render the component if the returning value changes. And `dayjs` I guess will be always creating a new instance of its class making the component re-rendering if `created` is the same

Reply:
And `dayjs(value)` can be applied later